conftest.py

```diff
@@ -0,0 +1,38 @@
+from pprint import pprint
```

Comment:
Let's remove this, not using pprint anywhere (also is not in the requirements file so will give error on running directly)

Reply:
Done https://github.com/browserstack/pytest-browserstack/pull/2/commits/998555687793bd1c39a34317c2a5b3d86319041a